app/resources/create.html.template

```diff
@@ -94,6 +115,14 @@
               </div>
             {% endif %}
 
+            <a href="#" class="identify-button" style="text-decoration:none">
+              {% trans "+ Other personal identify detail" %} <br>
+              <span class=hint>
+              {% trans "(Alternate name/ Home address/ Description/ Person's photo)" %}
```

Comment:
I believe it's more natural to put a whitespace before and after the slash e.g., "Alternate name / Home address ..."